Add more examples for BOOL aggregations to docs (#12566)
ydb-platform/ydb: ydb/docs/en/core/yql/reference/yql-core/builtins/_includes/aggregation/bool_bit.md

## BOOL_AND, BOOL_OR and BOOL_XOR {#bool-and-or-xor}

### Signature

```yql
BOOL_AND(Bool?)->Bool?
BOOL_OR(Bool?)->Bool?
BOOL_XOR(Bool?)->Bool?
```

Apply the relevant logical operation  (`AND`/`OR`/`XOR`) to all values in a Boolean column or expression.

> These functions **don't skip** `NULL` during aggregation and follows the next rule: `true and null == null`, `false or null == null`. For `BOOL_AND` with any `true` values and even one `NULL` the result is `NULL`, but even one `false` changes it to `false` regardless whether `NULL` is present. For `BOOL_OR` with any `false` valse and even one `NULL` the result is `NULL`, but even one `true` changes it to `true` regardless whether `NULL` is present. `BOOL_XOR` results to `NULL` if any `NULL` is found. One can find more examples of such behaviour below.

Reviewer: ```suggestion
Unlike most other aggregate functions, these functions **don't skip** `NULL` during aggregation and use the following rules: 

- `true AND null == null`
- `false OR null == null`

For `BOOL_AND`:

- If there is any `true` value and even one `NULL`, the result is `NULL`.
- If there is even one `false`, the result changes to `false`, regardless of whether `NULL` is present.

For `BOOL_OR`:

- If there is any `false` value and even one `NULL`, the result is `NULL`.
- If there is even one `true`, the result changes to `true`, regardless of whether `NULL` is present.

For `BOOL_XOR`:

- The result is `NULL` if any `NULL` is found.

Examples of such behavior can be found below.
```

Author: [Fixed](https://github.com/ydb-platform/ydb/pull/12566/commits/5291e5a09c3c91d3c9ec9c98b8cbb9b0dd3f51a3) along with /ru version.

> To skip `NULLs` during aggregation, you can use the functions `MIN`/`MAX` or `BIT_AND`/`BIT_OR`/`BIT_XOR`.

Reviewer: ```suggestion
To skip `NULL` values during aggregation, you can use the functions `MIN`/`MAX` or `BIT_AND`/`BIT_OR`/`BIT_XOR`.
```

Author: [Fixed](https://github.com/ydb-platform/ydb/pull/12566/commits/200750bf8893df78e95364c1a64fe9902d0827c2).

### Examples

```yql

> $data = AsList(
>     AsStruct(true  AS nonNull, true AS nonFalse, NULL  AS nonTrue, true  AS anyVal),
>     AsStruct(false AS nonNull, NULL AS nonFalse, NULL  AS nonTrue, NULL  AS anyVal),
>     AsStruct(false AS nonNull, NULL AS nonFalse, false AS nonTrue, false AS anyVal),
> );

Reviewer: ```suggestion
$data = [
    <|nonNull: true, nonFalse: true, nonTrue: NULL, anyVal: true|>,
    <|nonNull: false, nonFalse: NULL, nonTrue: NULL, anyVal: NULL|>,
    <|nonNull: false, nonFalse: NULL, nonTrue: false, anyVal: false|>,
];
```

Author: [Fixed](https://github.com/ydb-platform/ydb/pull/12566/commits/506b93bb6dfefd036f001e6eec102d4bf67dc62d).
